wordle-web/pages/ScoreStats.vue

```diff
@@ -43,7 +45,7 @@ export default class ScoreStats extends Vue {
   stats: any = []
 
   refreshStats() {
-    this.$axios.get('/api/ScoreStats').then((response) => {
+    this.$axios.get('https://wordle-api-3.azurewebsites.net/api/ScoreStats').then((response) => {
```

Comment:
I don't believe you need the base URL here because it should be set in the nuxt config file under the Axios setting. 

also, the relative path is calling the scorestats but your API only has leaderboard. Have axios get(/api/{API_METHOD_NAME}